### ISSUE TITLE
Rethink definition of displacement vector

### ISSUE THREAD

**Issue by sfmig** [neuroinformatics-unit/movement#136]
Right now the displacement vector at t is defined as the vector from t-1 to t. For the first frame, it is defined as the zero vector.

Although this naturally derives from the diff operator, it does feel a bit awkward for example when you want to visually check the displacement vector. Right now, you would typically use a quiver plot, in which at each position t of the trajectory you plot the _negative_ of the displacement vector at t. In this way, the resulting vector at t points to the previous position at t-1, and you can visually check the data behaves as you expect.

Plotting the _negative_displacement vector seems slightly cumbersome and adds to the mental load of things to remember about the data. It may show that a more useful definition of the displacement vector would be "the vector at t that goes from the position at t to the position at t+1". This would make for an easy visual check with quiver plot, and seems like an easier mental model to have of the data. For the last frame, we would define it as the zero vector.

**Comment by sfmig** [comment]
yes, I was aware of this option too 😁 My point is that it is a bit more intuitive to think displacement as t to t+1

**Comment by niksirbi** [comment]
This has been solved by #657.
